src/GlobalState.js

```diff
@@ -14,35 +14,44 @@ const GlobalContext = createContext({})
 export const GlobalConsumer = GlobalContext.Consumer
 
 let setProviderInstance
-const providerPromise = new Promise(resolve => { setProviderInstance = resolve })
+const providerPromise = new Promise(resolve => {
+  setProviderInstance = resolve
+})
 
 let setSignedIn
-const signInPromise = new Promise(resolve => { setSignedIn = resolve })
+const signInPromise = new Promise(resolve => {
+  setSignedIn = resolve
+})
 
 export const getProvider = () => providerPromise
 
 const AUTH = 'auth'
 const TOKEN_SECRET = 'kickback'
 const TOKEN_ALGORITHM = 'HS256'
 
-
 class Provider extends Component {
   state = {
     apolloClient: this.props.client,
     currentModal: null,
     auth: LocalStorage.getItem(AUTH) || {},
     networkState: {},
+    selectedFilter: null
+  }
+
+  handleFilterChange = selectedFilter => {
```

Comment:
Why is this in global state? filtering is only relevant on the single event page